addon/sidebar.html

```diff
@@ -17,7 +17,7 @@
       <h1 class="title">Welcome to Side View</h1>
       <h3 class="subtitle">A <a title=" Visit Test Pilot" href="https://testpilot.firefox.com">Firefox Test Pilot</a> Experiment</h3>
       <a class="default-link" href="#" title="Watch Tutorial">Watch Tutorial</a>
-      <a class="default-link" href="#" title="Give Feedback">Give Feedback</a>
+      <a class="default-link" href="https://qsurvey.mozilla.com/s3/Test-Pilot-Side-View?ref=sidebar" href="#" title="Give Feedback">Give Feedback</a>
```

Comment:
This looks suspicious to my eyes. I think we now have two `href=""` attributes, with conflicting values.

Reply:
Should be fixed!